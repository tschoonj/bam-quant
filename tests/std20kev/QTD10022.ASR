File: C:\xrfdaten\2014KW30\chris\std20kev\std10022.spe
Start: 21.07.2014 17:59:59
Stop: 21.07.2014 18:01:39
BESSY: 298.9212427 298.9375987
IONI: 3.941771E-11 3.953389E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 93670
Run: 100000
Dead: 6
ICR: 15040
OCR: 0
Accumulated BESSY: 59706.3109667
Accumulated Ioni: 7.88680728E-09
N Accumulation: 200
Normfactor:  2.7072493e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ti
exp_x 71.2603988647461 71.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  22   1      4.509   1176804.      1085.      63.19
