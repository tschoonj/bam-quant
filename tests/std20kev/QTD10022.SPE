$DATA:
0 2000
21804
14024
5918
1681
373
46
8
1
2
0
0
0
1
1
0
0
76
337
400
403
366
383
385
373
373
340
354
332
312
342
291
300
286
288
285
298
286
303
243
268
247
239
247
230
229
258
279
270
233
235
212
211
211
255
221
232
224
220
249
242
227
230
227
220
196
212
203
207
198
206
219
218
249
227
223
235
237
201
209
217
196
198
211
220
188
198
209
205
202
206
184
210
229
215
221
210
257
189
217
240
241
273
257
254
237
228
254
199
216
238
198
213
214
225
220
223
213
199
231
221
204
244
212
237
238
227
251
261
235
289
308
358
495
679
940
1263
1546
1517
1314
1116
764
539
397
355
394
422
454
432
376
360
326
267
268
309
328
352
442
466
442
402
356
319
302
267
272
286
248
294
282
287
278
311
290
317
303
292
305
285
332
342
312
365
365
375
378
369
358
386
395
429
421
405
430
455
491
458
484
471
478
496
467
499
509
555
515
557
574
650
720
745
870
973
1136
1616
2123
3591
7244
14637
29342
53438
87451
125211
156521
170595
163555
136484
99020
62883
35330
17543
7867
3421
1539
1157
1008
1186
1630
2345
3813
6303
10381
15551
20682
24926
26494
24659
20646
14917
9675
5547
2654
1240
508
167
52
39
18
13
13
7
18
9
10
12
8
8
11
12
16
6
13
7
10
10
4
9
5
8
9
4
5
7
6
10
6
4
4
2
7
10
10
11
10
14
8
10
2
9
6
6
1
6
9
8
6
8
9
7
9
13
10
11
12
11
32
23
21
33
19
15
11
16
15
6
9
9
11
8
8
7
5
6
4
4
5
7
6
9
19
14
25
34
27
28
30
19
15
19
23
13
9
14
7
12
7
7
8
16
11
8
7
16
9
29
31
36
57
55
103
99
122
111
92
76
49
26
27
15
12
11
10
8
6
7
9
13
11
16
15
9
14
21
13
8
14
8
15
13
8
8
10
8
12
13
12
13
14
19
26
30
33
31
26
40
34
25
21
34
20
13
21
5
10
14
10
15
13
10
9
12
11
15
8
9
8
13
10
22
21
24
66
63
95
127
137
149
182
161
149
128
100
72
47
32
26
15
16
9
22
16
21
24
36
28
47
56
56
64
65
51
47
35
37
28
21
20
27
22
16
22
25
29
25
20
11
20
10
15
22
10
14
16
12
10
7
14
15
7
7
8
9
7
4
12
7
5
9
6
6
4
4
3
9
5
9
6
15
10
16
10
9
13
16
18
22
12
13
8
7
9
9
4
10
3
6
5
8
7
2
8
4
7
10
5
6
9
6
8
9
5
9
5
5
10
5
4
9
9
6
6
9
3
4
9
5
6
6
2
12
7
5
2
5
12
7
6
8
5
3
7
3
6
6
6
4
2
4
4
6
1
6
9
7
5
5
5
8
5
3
4
7
3
3
7
2
5
4
3
10
5
5
1
6
7
5
6
6
2
6
4
8
5
11
6
10
11
8
7
13
5
6
3
10
6
4
3
2
7
1
5
5
6
4
3
8
6
5
2
1
4
3
4
3
4
4
4
5
4
3
3
2
4
2
2
5
2
4
3
4
6
4
5
3
4
5
5
6
4
5
4
0
3
1
5
3
6
5
4
2
4
2
7
4
2
2
1
2
5
1
3
3
8
3
3
4
1
6
6
3
2
5
3
1
4
5
4
2
1
1
4
4
6
2
2
4
4
5
2
4
2
3
4
0
3
5
4
2
4
0
1
2
4
3
2
1
3
0
2
3
3
3
2
2
1
0
3
2
2
1
3
3
1
0
3
5
4
3
4
6
10
2
10
14
20
25
31
40
46
42
45
39
47
42
45
33
23
27
18
9
11
10
8
6
2
3
6
1
0
0
3
5
3
4
2
1
3
4
1
4
3
0
3
2
1
1
1
3
6
2
5
2
2
4
4
3
4
4
3
5
4
2
3
6
2
4
5
2
4
2
0
3
4
5
4
4
5
2
5
1
5
3
1
4
4
5
5
1
6
5
5
5
5
7
4
3
6
8
7
7
12
11
13
12
13
11
12
16
6
12
9
6
6
10
9
12
9
6
7
13
11
6
11
7
12
12
10
11
9
5
7
13
9
16
15
11
16
8
11
17
18
10
11
23
14
15
18
13
15
21
20
16
21
18
26
24
27
32
22
30
26
27
40
46
33
37
41
37
44
42
32
48
43
47
54
57
64
61
61
72
67
61
70
79
63
73
91
70
68
69
64
65
62
72
88
68
87
76
81
77
73
72
83
75
93
83
91
106
117
96
139
125
151
176
179
213
221
206
269
243
250
269
247
209
202
238
211
183
156
165
130
131
106
105
74
56
45
37
41
16
21
16
15
9
2
4
2
2
0
0
3
0
0
2
1
0
1
1
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
2
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
1
0
1
0
0
0
0
0
2
0
0
0
1
0
0
0
2
1
0
1
2
0
1
0
0
0
1
1
1
0
1
0
0
0
0
1
1
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10022.spe
Start: 21.07.2014 17:59:59
Stop: 21.07.2014 18:01:39
BESSY: 298.9212427 298.9375987
IONI: 3.941771E-11 3.953389E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 93670
Run: 100000
Dead: 6
ICR: 15040
OCR: 0
Accumulated BESSY: 59706.3109667
Accumulated Ioni: 7.88680728E-09
N Accumulation: 200
Normfactor:  2.7072493e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ti
 
exp_x 71.2603988647461 71.2604
exp_y 150.323791503906 150.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  22   1      4.509   1176804.      1085.      63.19
