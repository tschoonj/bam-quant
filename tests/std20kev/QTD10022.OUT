     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:36  บ   
     บ  Spectrum: QTD10022.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =    33.3   last change =   .06%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -53.1 ๑   100.0   บ     -53.1 ๑     1.3   บ   
     บ  GAIN (eV/ch)  บ     20.355 ๑   2.000   บ    20.355 ๑    .006   บ   
     บ det NOISE (eV) บ      -55.6 ๑    40.0   บ     -55.6 ๑    -8.6   บ   
     บ  FANO factor   บ       .218 ๑    .050   บ      .218 ๑    .007   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ti-Ka                           1176804. ๑   1085.           บ   
     บ    KA1         4.509   1.00000     1176804. ๑   1085.     63.2  บ   
     บ              224.125    133.87         139.           7.84E-01  บ   
     บ                                                                 บ   
     บ  2 Ti-Kb                            191712. ๑    438.           บ   
     บ    KB1         4.932   1.00000      191712. ๑    438.     29.7  บ   
     บ              244.906    141.04         132.           8.30E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                            1904. ๑     47.           บ   
     บ    L1N3        9.018    .73941        1408. ๑     37.     12.3  บ   
     บ              445.643    197.33         115.           9.69E-01  บ   
     บ    L1N2        9.441    .24095         459. ๑     15.      3.8  บ   
     บ              466.424    202.26         108.           9.73E-01  บ   
     บ    L1O3        9.864    .01963          37. ๑     10.      6.3  บ   
     บ              487.206    207.08         102.           9.76E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
